spec/marqeta/api_object_spec.rb

```diff
@@ -80,31 +80,90 @@
     end
 
     describe '.object_list' do
-      let(:response_hash) { { 'data' => data_array } }
-      let(:data_array) do
-        [
-          { token: 'token1' },
-          { token: 'token2' }
-        ]
-      end
       let(:endpoint) { 'endpoint' }
-      let(:api_caller) { instance_double(Marqeta::ApiCaller) }
-
-      before do
-        allow(Marqeta::ApiCaller)
-          .to(receive(:new))
-          .with(endpoint)
-          .and_return(api_caller)
+      let(:query_results_count) { Marqeta::ApiObject::QUERY_RESULTS_COUNT }
+
+      context 'when api returns no results' do
+        let(:response_hash) { { 'data' => [], 'is_more' => false } }
+        let(:paginated_endpoint) { "#{endpoint}?count=#{query_results_count}&start_index=0" }
```

Comment:
I think it may even be more clear to just use the raw number "100" in the tests. But whatever you think.

Reply:
@dubroe ah, trying to keep test different from implementation. will do!